storage/badger/cache.go

```diff
@@ -94,12 +94,14 @@ func (c *Cache) Get(key interface{}) func(*badger.Txn) (interface{}, error) {
 		}
 
 		// get it from the database
-		c.metrics.CacheMiss(c.resource)
```

Comment:
There are 2 cases when cache was missed:
1. the queried item was not found in database
2. the queried item exists in database.

We now use different metrics to distinguish them